src/central-submissions.rst

```diff
@@ -143,3 +147,52 @@ To connect with Excel or Power BI, follow these steps.
 
 If you want to use the free and popular `R statistics and analysis tool <https://www.r-project.org/>`_, we recommend you use `ruODK <https://docs.ropensci.org/ruODK/>`_. A guide for getting started with it can be found `here <https://docs.ropensci.org/ruODK/articles/odata-api.html>`_. ruODK is developed and supported by community members. If you wish to help improve it, you can find information `on GitHub <https://docs.ropensci.org/ruODK/CONTRIBUTING.html>`_.
 
+.. tip::
```

Comment:
I guess this is helpful, but feels a little weird since it's probably not a common problem? I dunno.

Reply:
this edit was a [suggestion from @florianm](https://forum.getodk.org/t/odk-central-odata-refused-connection-to-powerbi-and-excel/20260/30?u=issa)

Reply:
I've definitely run into it before and it was a pain to figure out that I could clear credentials and a cache. What do you think @lognaturel? Have you had to do this before?

Reply:
Haven't had to do it. I think it's useful to add but maybe would be most likely to be found in the troubleshooting section? This seems like a reasonable place for it but given that it's rare, I wonder if users would know to come back here.

Reply:
(shrug) that's kind of my feeling on the docs in general; how much of these get read and how often? anyway doesn't sound like anyone has too much of a problem with it being here, and the troubleshooting  section we have is in the setting up section and feels aimed  at administrators, not end users, so i think i'll just leave it here.

Reply:
"you probably entered the wrong password" doesn't capture what likely happened? I think it's worth explicitly saying that one way this happens is if a user changes their Central password. That should help users search for it and/or anticipate the case if it happens again.

Reply:
no. it doesn't. part of the problem is that powerbi is imprecise and possibly buggy about reusing passwords. so often the problem will come up the very first time you even attempt to log into an instance over pbi.